.github/workflows/nightly_release.yml

```diff
@@ -0,0 +1,67 @@
+name: SalesforceCommerceCloud/pwa-kit/nightly_release
+on:
+  workflow_dispatch:
+  schedule:
+    # Run every day at 12am (PST) - cron uses UTC times
+    - cron: "0 8 * * *"
+
+jobs:
+  create_nightly_release:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
```

Comment:
I assume we need to write access because we want to create `release-*` branches.

Is it the best approach to create a new release- branch per nightly release?